lib/main.dart

```diff
@@ -1,328 +1,17 @@
+import 'package:count_habits/presentation/pages/app.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter/services.dart';
 
 void main() {
-  runApp(const App());
+  runApp(const Habits365());
 }
 
-class App extends StatelessWidget {
-  const App({super.key});
+class Habits365 extends StatelessWidget {
+  const Habits365({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
-      home: MyWidget(),
+    return MaterialApp(
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kolbe-ryo/count_habits/blob/bc079e0f667d8943dac55602b8911510408a88ef/lib/main.dart#L13"></span>Use 'const' with the constructor to improve performance</td>
    </tr>
  </tbody>
</table>